### PR TITLE
Update abapGit-installation.md

### PR DESCRIPTION
**Pull request by AndreaBorgia-Abo**
naming suggestions for demos package and explanation

### CODE REVIEW

**Comment by AndreaBorgia-Abo** [comment]
Back to draft to fix a typo and, also, to force abaplint checks to restart (somehow it got stuck)
@larshp any tips?

**Comment by larshp** [comment]
push another commit, abaplint was down, see https://status.abaplint.app (Kubernetes argh 😠 )

**Comment by larshp** [comment]
updated, results as expected 😄

File: docs/abapGit-installation.md [hunk 1]
```diff
@@ -11,3 +11,5 @@ Execute abapGit using the report **ZABAPGIT_FULL**, click on *New Online*, fill
 To install abap2xlsx click *pull*.
 
 After installing, if you want to install the **demo programs**, repeat the actions above, starting from *New Online* to *pull*, and fill *Git repository URL* with *https://github.com/abap2xlsx/demos*.
+
+Be careful with the package name for demos: it should NOT be the basename of your abap2xlsx installation followed by "_demos"! If you have ZABAP2XLSX as base package, you may use ZABAP2XLSXDEMOS but not ZABAP2XLSX_DEMOS, for example. Quick background for the curious folks: before demos were spun off as a separate repository, they were placed in the "demos" subpackage of abap2xlsx and, yes, abapGit encodes the package / subpackage relationship with an underscore.
```
Comment (larshp) [inline docs/abapGit-installation.md:15]: well, suggest encouraging developers to install the demos in a local package, as demo programs are not needed in production

**Review by AndreaBorgia-Abo — COMMENTED** [review]
added suggested changes

**Review by larshp — APPROVED** [review]
lgtm

**Comment by AndreaBorgia-Abo** [comment]
Followup notes for @larshp : when squashing multiple commits, feel free to edit the logs accordingly.
For example in this case the logs could have been shortened to this:
```
* Update abapGit-installation.md
naming suggestions for demos package and explanation
recommend that demos be created as local package
```

**Pull request merged**